File: Program69.cpp
#include<iostream>
using namespace std;

class ArrayX
{
    private:
        int *Arr;
        int iSize;

    public:
        ArrayX(int iValue)              
        {
            this->iSize = iValue;
            Arr = new int[iSize];     
        }

        ~ArrayX()                     
        {
            delete []Arr;               
        }

        void Accept()   
        {
            int iCnt = 0;
            cout<<"Enter the elements"<<endl;
            for(iCnt = 0; iCnt < iSize; iCnt++)
            {
              cin>>Arr[iCnt];
            }
        }
        // Positive
        int Maximum()
        {
            int iCnt = 0;
            int iMax = 0;

            for(iCnt = 0; iCnt< iSize; iCnt++)
            {
               if(iMax < Arr[iCnt])
               {
                   iMax = Arr[iCnt];
               }
            }
            return iMax;
        }

};

int main()
{
    int iValue = 0;
    int iRet = 0;

    cout<<"Enter the number of elements:"<<endl;
    cin>>iValue;

    ArrayX obj1(iValue);

    obj1.Accept();      
    iRet = obj1.Maximum();     
    cout<<"Maximum number is: "<<iRet<<endl;
    
    return 0;
}

File: Program69.java
import java.lang.*;
import java.util.*;

class ArrayX
{
    private int Arr[];

    public ArrayX(int iNo)
    {
        Arr = new int[iNo];
    }

    public void Accept()
    {
        int iCnt = 0;

        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the values : ");

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }
    }
    // Positive
    public int Maximum()
    {
        int iCnt = 0;
        int iMax= 0;

        for(iCnt = 0; iCnt< Arr.length; iCnt++)
        {
            if(iMax < Arr[iCnt])
            {
                iMax = Arr[iCnt];
            }
        }
        return iMax;
    }
}

class Program69
{
    public static void main(String arg[])
    {
            Scanner sobj = new Scanner(System.in);
            int iLength = 0, iRet =0;

            System.out.println("Enter the value for constructor");
            iLength = sobj.nextInt();

            ArrayX obj = new ArrayX(iLength);

            obj.Accept();
            iRet = obj.Maximum();
            System.out.println("Maximum number is: "+iRet);
            
            obj = null;
    }
}